### PR TITLE
Adding look for logs on all hosts

### PR DESCRIPTION
**Pull request by igal-stratoscale**
Adding --all-nodes option to bring logs from all nodes
@Stratoscale/systegration

### CODE REVIEW

File: py/strato/common/log/log2text.py [hunk 1]
```diff
@@ -470,6 +494,8 @@ def minimumLevel(minLevel, noDebug):
     parser.add_argument("-t", "--tail", type=int, metavar='n', help='print n last lines only', default=0)
     parser.add_argument("--since", type=str, metavar='DATE', help='Show entries not older than the specified date (e.g., 1h, 5m, two hours ago, 8/aug/1997)')
     parser.add_argument("--until", type=str, metavar='DATE', help='Show entries not newer than the specified date (e.g., 0.5h, 4m, one hour ago)', default="01/01/2025")
+    parser.add_argument("--all-nodes", action='store_true', help='Bring asked logs from all nodes and open them')
+    parser.add_argument("--cached", action='store_true', help='Find logs in /var/log/inspector/strato_log')
```
Comment (rom-stratoscale) [inline py/strato/common/log/log2text.py]: help: used previously collected logs, placed in /var/log/inspector/strato_log

File: py/strato/common/log/log2text.py [hunk 2]
```diff
@@ -425,6 +425,30 @@ def executeRemotely(args):
...
+        command = ["inspector", "log-dump"]
+        command.extend(args.logFiles)
+        command.extend(["-q", "-u", args.user, "-p", args.password, '--clear-dst-folder'])
+        try:
+            output = subprocess.check_output(command, stderr=subprocess.STDOUT, stdin=open('/dev/null'), close_fds=True)
+        except subprocess.CalledProcessError as exc:
+            print "Failed in copying files from remotes. Command : %s\n Return Code : %s\n Exception : %s" % \
+                  (command, exc.returncode, exc.output)
+            raise
+        logFiles = output.splitlines()
```
Comment (rom-stratoscale) [inline py/strato/common/log/log2text.py]: what's the output is?

File: py/strato/common/log/log2text.py [hunk 3]
```diff
@@ -425,6 +425,30 @@ def executeRemotely(args):
     return runRemotely(args.node, ignoreArgs)
 
 
+def copy_log_files_from_remotes(args):
+    if not args.cached:
+        if not args.user or not args.password:
+            raise Exception("Please provide user and password")
+        command = ["inspector", "log-dump"]
```
Comment (rom-stratoscale) [inline py/strato/common/log/log2text.py]: full path maybe?

File: py/strato/common/log/log2text.py (hunk 2 above)
Comment (igal-stratoscale) [inline py/strato/common/log/log2text.py]: file paths that inspector brought

File: py/strato/common/log/log2text.py (hunk 3 above)
Comment (igal-stratoscale) [inline py/strato/common/log/log2text.py]: usr/bin? yap , why not. Sure will be better

**Pull request merged**